src/components/codeBlock/code-blocks.module.scss

```diff
@@ -73,12 +76,16 @@
   :global(.diff-inserted),
   :global(.inserted) {
     background-color: rgba(16, 185, 129, 0.2);
+    margin-left: -12px;
+    width: calc(100% + 12px);
   }
 
   /* Set deleted line (-) color */
```

Comment:
Maybe update this comment (and the comment for the above changes) to reflect that we are also setting the padding here